blocks/footer/footer.css

```
@@ -16,5 +17,134 @@ footer .footer p {
 
 footer .footer-slogan {
   font-size: var(--footer-font-size-slogan);
+  padding-bottom: 40px;
+  font-style: italic;
+}
+
+footer li {
+  display: inline-block;
+  margin-right: 1rem;
+}
+
+footer a {
+  color: #fff;
+}
+
+footer li a:any-link {
+  color: white;
+}
```

let's use the css variables. 
```suggestion
footer li a:any-link {
  color: var(--text-color-on-black-background);
}

footer li a:hover {
  color: var(--link-hover-color);
}
```